File: Beecrowd/1585 Fazendo Pandorgas/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int N, i, x, y, cm;

    cin >> N;

    for (i = 0; i < N; i++)
    {
        cin >> x >> y;
        cm = x * y / 2;

        cout << cm << " cm2" << endl;
    }

    return 0;
}

File: Beecrowd/1585 Fazendo Pandorgas/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        StringBuilder sb = new StringBuilder();

        int n, x, y, cm;

        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            y = sc.nextInt();

            cm = x * y / 2;

            sb.append(cm + " cm2\n");
        }

        System.out.print(sb);

        sc.close();
    }

}